src/facade/op_status.cc

```diff
@@ -0,0 +1,46 @@
+#include "facade/op_status.h"
+
+namespace facade {
+
+const char* ToString(OpStatus op) {
```

Comment:
Where it is used? only for debugging?

Reply:
I was missing this in the log, where you want to see the actual enum name and not the number, so it would be outputted now, when you're "stream out" the value of the status from OpResult

Reply:
it's fine but a bit confusing given that we also output it via ReplyBuilder (SendError routing). 
Therefore I suggest to call it `DebugString` and not `ToString` to make it more clear that's it's not for prod use

Reply:
I will reame it to DebugString